### ISSUE TITLE
adjusted p-values with facets

### ISSUE THREAD

**Issue by choc2000** [kassambara/survminer#407]
Hi,

I was not able to identify a built-in way to adjust p-values for multiple testing when using faceted ggsurvplot.

Current workaround is to pre-compute adjusted p-values for each subgroup and add them separately to the resulting facets using an additional geom_text layer.

Wouldn't it make sense to have the option to apply p-value correction to faceted plots?

**Comment by kassambara** [comment]
Good point — and to be clear, this is a genuine gap: `ggsurvplot_facet(..., pval = TRUE)` computes a per-panel p-value but does **not** adjust it across panels (the displayed p-values are raw). There's no built-in `p.adjust.method` for the facet path yet, so keeping this open as the enhancement (it should mirror `pairwise_survdiff()`, which already has `p.adjust.method`).

Interim, the manual approach (as you noted) works — compute per-facet p, adjust across facets, add a `geom_text` layer:

```r
library(survminer); library(survival); library(ggplot2)
d <- subset(colon, etype == 2); d$sex <- factor(d$sex, labels = c("F", "M"))
sexes <- levels(d$sex)
padj <- do.call(rbind, lapply(sexes, function(s) {
  sd <- survdiff(Surv(time, status) ~ rx, data = d[d$sex == s, ])
  data.frame(sex = factor(s, levels = sexes),
             pval = pchisq(sd$chisq, length(sd$n) - 1, lower.tail = FALSE))
}))
padj$label <- paste0("adj.p = ", signif(p.adjust(padj$pval, "BH"), 2))

ggsurvplot_facet(survfit(Surv(time, status) ~ rx, data = d), data = d,
                 facet.by = "sex", pval = FALSE) +
  geom_text(data = padj, aes(x = Inf, y = Inf, label = label),
            hjust = 1.1, vjust = 1.5, inherit.aes = FALSE, size = 3.3)
```

![per-facet BH-adjusted p-values](https://i.imgur.com/UXxVLTr.png)

But you shouldn't have to hand-roll it — tracking a native `p.adjust.method` argument for `ggsurvplot_facet()` here.

**Comment by kassambara** [comment]
Added in the dev version (0.5.2.999): `ggsurvplot_facet()` gains a `p.adjust.method` argument that adjusts the per-panel log-rank p-values for multiple comparisons across panels (passed to `stats::p.adjust()`), mirroring `pairwise_survdiff()`. The default `"none"` keeps the raw per-panel p-values; when a method is set the displayed text is prefixed `adj.p =`.

```r
# remotes::install_github("kassambara/survminer")
library(survminer); library(survival)
d <- subset(colon, etype == 2); d$sex <- factor(d$sex, labels = c("F", "M"))
fit <- survfit(Surv(time, status) ~ rx, data = d)
ggsurvplot_facet(fit, data = d, facet.by = "sex", pval = TRUE, p.adjust.method = "BH")
```

![per-panel p-values, BH-adjusted across panels](https://i.imgur.com/3jk8NWD.png)

Panels with an undefined p-value (e.g. a single group) stay blank and are excluded from the adjustment count. Thanks @choc2000.

**Comment by kassambara** [comment]
Follow-up: the adjusted-p-value label is now customisable via a `p.adjust.label` argument (default `"adj.p ="`, so existing output is unchanged). Handy for publication styling.

```r
library(survminer); library(survival)
d <- subset(colon, etype == 2); d$sex <- factor(d$sex, labels = c("F", "M"))
fit <- survfit(Surv(time, status) ~ rx, data = d)

# default prefix
ggsurvplot_facet(fit, data = d, facet.by = "sex", pval = TRUE, p.adjust.method = "BH")
#> panels: "adj.p = 0.62"  "adj.p = 0.00029"

# custom prefix
ggsurvplot_facet(fit, data = d, facet.by = "sex", pval = TRUE,
                 p.adjust.method = "BH", p.adjust.label = "q =")
#> panels: "q = 0.62"  "q = 0.00029"
```

For a very small p-value the label's trailing `=` becomes `<` (e.g. `"adj.p < 0.0001"`).
